documentation/wtpdf-from-latex.md

```diff
@@ -0,0 +1,251 @@
+% Classic LaTeX document markup from here.
+\begin{document}
+
+\section{A small example}
+
+Some inline mathematics involving $x\in\symbb{R}$.
+
+A list of items
+\begin{enumerate}
+\item a first item;
+\item a second item;
+\item a third item.
+\end{enumerate}
+
+Display math
+\[
+c=\sqrt{a^2+b^2}
+\]
+
+\end{document}
+```
+
+
+----
+
+You may directly run the example above, or edit it within this web
+page, before generating the PDF.  Please keep the examples
+relatively small.
+
+The PDF will (by default) be generated via a standard TeXLive 2024
+  lualatex-dev. The options used in the above example will tag every
+  math expression with the TeX source of the expression, and with an
+  automatically generated MathML version.  The resulting tagging, and
+  the derived HTML (using the MathML associated with each math
+  formula) may be viewed by following the link to the ngPDF site.
+
+
+Initially the example will use a MathML Structure Element
+tagging. If you remove the configuration line where indcated, then
+LaTeX will instead associate each math expression with
+two associated files, one with the TeX source, and one with generated MathML.
+
+
+The submitted TeX source is deleted as soon as the PDF is
+generated, and the resulting PDF is deleted after an hour.
+
+
+
+----
+
+The above small example shows the basic features of a Tagged LaTeX document.
+Here we show a larger more complex two-column document including generated
+cross references and citations. Some familiarity with LaTeX is assumed, and
+we do not show this by default.
+
+
+
+<details markdown="1">
+
+<summary><h3 style="display:inline">A Larger Example</h3></summary>
+
+
+<p> </p>
+
+```latex
+\DocumentMetadata{
+  lang        = en,
+  pdfversion  = 2.0,
+  pdfstandard = ua-2,
+  pdfstandard = a-4f,
+  testphase   = latest
+}
+
+% Remove this line to generate MathML Associated Files rather than Structure Elements
+% The file would then need to declare PDF/A-4f rather than PDF/A-4 conformance.
+\tagpdfsetup{math/mathml/structelem, math/mathml/AF=false, math/tex/AF=false, math/mathml/sources=}
+
+\documentclass[a4paper,twocolumn]{article}
+
+\usepackage{unicode-math}
+\usepackage{graphicx}
+
+\begin{document}
+
+\title{A Larger Example}
+\date{2024-12-17}
+\author{The \LaTeX\ Team}
+
+\maketitle
+
+\begin{abstract}
+  An example document showing automatic tagging of typical structures
+  found in a \LaTeX\ document, including titles, sections, lists,
+  citations lists and mathematics. A two column layout is used, the
+  tagging enables the reading order to correctly follow the flow of
+  text through the columns.
+\end{abstract}
+
+\tableofcontents
+
+\section{Introduction}
+
+This document shows a typical two-column document incorporating tables, figures and mathematics.
+
+Apart from two commands at the start to enable tagging, and a small
+amount of additional markup to give alternative texts for graphics
+inclusion, and to specify the heading rows of tables. The document
```

Comment:
we should have a footnote or two as part of the document.

one could go here to explain that the table syntax inside the document is not exactly how the final product will do it
(or maybe not but somefootnotes as such would be good imho